cinema/views.py

```diff
@@ -43,6 +60,21 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        title = self.request.query_params.get("title", None)
+        genres = self.request.query_params.get("genres", None)
+        actors = self.request.query_params.get("actors", None)
+
+        if genres:
+            genres_ids = [int(genre_id) for genre_id in genres.split(",")]
```

Comment:
DRY 

here and in 74 line
create small method `params_to_int` for it

Reply:
Still not fixed